src/button.test.forms.ts

```diff
@@ -0,0 +1,144 @@
+import { html } from 'lit/static-html.js';
+import { expect, test } from './playwright/test.js';
+
+test(
+  'has a `form` property that returns its form',
+  { tag: '@forms' },
+  async ({ mount, page }) => {
+    await mount(html`
+      <form>
+        <glide-core-button label="Label"></glide-core-button>
+      </form>
```

Comment:
An advantage our `mount()` has over `fixture()` from `@open-wc/testing` is that you don't need to provide a [`parentNode`](https://open-wc.org/docs/testing/helpers/#customize-the-fixture-container) option whenever you want a to wrap a component with another element. 

You can render whatever you need to.